CONTRIBUTE.md

```diff
@@ -33,4 +33,157 @@ def get_context(self, request: WSGIRequest) -> dict:
 like this
 please add this in the context of the view because this loads all the pfp and permissions
 
+## directory structure
+The following is the tree representation of the code jam website
+```
+.
+├── CONTRIBUTE.md
```

Comment:
Why not link a file, this is probably not looking good in the Contributing guide.

Reply:
Done. See below for more info